notifications/01-07-22-1300.md

```diff
@@ -0,0 +1,31 @@
+**What is happening?**
+
+Currently network traffic leaving the OpenShift clusters is mapped to a single NAT IP. This will be expanded to 4 IPs used in round robin. Existing firewall rules will be updated to match the new set of IPs, which will allow the existing firewall fules for **all existing connections between the Openshift clusters and other components within the BC Government network to continue functioning without any other changes required**.
```

Comment:
```suggestion
Currently network traffic leaving the OpenShift clusters is mapped to a single NAT IP. This will be expanded to 4 IPs used in round robin. Existing firewall rules will be updated to match the new set of IPs, which will allow the existing firewall rules for **all existing connections between the Openshift clusters and other components within the BC Government network to continue functioning without any other changes required**.
```